libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/contract/ManeuversContract.kt

```diff
@@ -0,0 +1,39 @@
+package com.mapbox.navigation.dropin.contract
+
+import com.mapbox.bindgen.Expected
+import com.mapbox.navigation.base.trip.model.RouteProgress
+import com.mapbox.navigation.dropin.processor.ManeuversProcessor
+import com.mapbox.navigation.dropin.state.NavigationState
+import com.mapbox.navigation.ui.maneuver.api.MapboxManeuverApi
+import com.mapbox.navigation.ui.maneuver.model.Maneuver
+import com.mapbox.navigation.ui.maneuver.model.ManeuverError
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.map
+
+sealed class ManeuversAction {
```

Comment:
This shouldn't be an action that developers have access to. This is an internal logic (and method of transferring data), so this and all other classes below should all be internal.